### ISSUE TITLE
ERROR: Cannot read property 'map' of undefined

### ISSUE THREAD

**Issue by hagope** [hacksalot/HackMyResume#82]
```
bc@debian:~/myresume$ hackmyresume BUILD resume.json TO out/resume.all

*** HackMyResume v1.3.1 ***
Applying MODERN theme (5 formats)
Reading resume: resume.json
ERROR: Cannot read property 'map' of undefined
TypeError: Cannot read property 'map' of undefined
    at skillsToFRESH (/usr/lib/node_modules/hackmyresume/src/core/convert.js:361:19)
    at Object.module.exports.toFRESH (/usr/lib/node_modules/hackmyresume/src/core/convert.js:60:17)
    at Object.module.exports.loadOne (/usr/lib/node_modules/hackmyresume/src/core/resume-factory.js:62:64)
    at /usr/lib/node_modules/hackmyresume/src/core/resume-factory.js:35:21
    at Array.map (native)
    at Object.module.exports.load (/usr/lib/node_modules/hackmyresume/src/core/resume-factory.js:34:22)
    at build (/usr/lib/node_modules/hackmyresume/src/verbs/generate.js:61:32)
    at main (/usr/lib/node_modules/hackmyresume/src/index.js:79:42)
    at Object.<anonymous> (/usr/lib/node_modules/hackmyresume/src/index.js:27:3)
    at Module._compile (module.js:460:26)
```

**Comment by hacksalot** [comment]
Hi @hagope thanks for the report -- will be fixed in the next version. Does running `hackmyresume validate resume.json` show an error?

**Comment by hagope** [comment]
Nope, says "Valid"

**Comment by NegatioN** [comment]
Are there any workarounds for this?

**Comment by hacksalot** [comment]
@hagope Does your source resume have a `skills` section? How is it structured?

@NegatioN If you're getting that specific error, and you don't have a skills section, try adding one. 1.4 will be out shortly which should resolve this and most "missing field" errors.

**Comment by NegatioN** [comment]
Good to hear. I do have a skills-section included already though.

    "skills": [
      {
      "name": "Teknologier",
      "level": "test",
      "keywords": [
        "Apache Kafka",
        "Kubernetes",
        "Docker",
        "test"
      ]
    }]
And i'm getting a valid response.

    *** HackMyResume v1.3.1 ***
    Validating resume.json against JSON RESUME schema: VALID!

**Comment by hacksalot** [comment]
Should be fixed in latest (v1.4.0).
